appIcons.js

```diff
@@ -1741,59 +1741,82 @@ var MyShowAppsIconMenu = Utils.defineClass({
     _dtpRedisplay: function() {
         this.removeAll();
 
-        if (this.sourceActor != Main.layoutManager.dummyCursor) { 
-            let powerSettingsMenuItem = this._appendMenuItem(_('Power options'));
-            powerSettingsMenuItem.connect('activate', function () {
-                Util.spawn(['gnome-control-center', 'power']);
+        // true if `which` can find the app
+        function _checkExists(app) {
```

Comment:
That seems like a handy function, would you mind moving it to the utils.js file and remove the preceding underscore in its name?

Reply:
No problem! I'll have an update as soon as I finish testing. Thanks!